### DISCUSSION TITLE
About training python

### DISCUSSION THREAD

**Discussion by minchaoyue** [snakers4/silero-vad#44] [Q&A] [answered]
Do you plan to release training code (python) ?  
or where get it ?

**Comment by snakers4** [comment, marked as answer]
Hi,

The short answer is - no / not yet.

The long answer is - we position our vad as a low level solution for end users to integrate into their apps (especially edge apps), not as framework / toolkit to build your own models.

If you have any feature requests - you are welcome, we aggregate the community feedback to prioritize next releases and / or quick patches.
